epub33/core/index.html

```diff
@@ -1303,8 +1154,8 @@ <h5>Shared Attributes</h5>
 								<p>An absolute or relative IRI reference [[!RFC3987]] to a resource.</p>
 								<div class="example">
 									<pre>&lt;link rel="record"
-      href="meta/9780000000001.xml" 
-      media-type="application/marc"/&gt;</pre>
+		href="meta/9780000000001.xml" 
```

Comment:
Looks like your editor is reformatting spacing in examples to tabs. Throws the alignment off in browsers.

Reply:
Sigh. Spaces are exchanged against tabs...

It should be o.k. now, but better check on your side, too.